superdesk/core/resources/model.py

```diff
@@ -110,6 +110,26 @@ def model_validate(
             obj["_type"] = item_type
         return instance
 
+    def to_dict(self) -> dict[str, Any]:
```

Comment:
Might be good to allow providing options, such as [use_objectid](https://github.com/superdesk/superdesk-core/blob/async/superdesk/core/resources/service.py#L257). This would allow us to define supported context attributes.

If any options are defined, then the `context` kwarg would be provided to the `model_dump` call.

Reply:
Done!